### PR TITLE
Update mix.en.md

### PR DESCRIPTION
**Pull request by lmhmhl**
Fixes #ISSUSE_ID.

Changes proposed in this pull request:
-
-
-

### CODE REVIEW

File: docs/document/content/user-manual/shardingsphere-jdbc/configuration/yaml/mix.en.md [hunk 1]
```diff
@@ -3,4 +3,67 @@ title = "Mixed Rules"
...
+* Configuration Item Explanation
+
+## Configuration Item Explanation
+```yml
+
+rules:
+  - !SHARDING
+    tables:
+      t_order_item_calcite_sharding:
+        actualDataNodes: calcite_ds.t_order_item_calcite_sharding_${0..1}
```
Comment (tristaZero) [inline docs/document/content/user-manual/shardingsphere-jdbc/configuration/yaml/mix.en.md:15]: 1. Could you give `calcite_ds` some explanation?

File: docs/document/content/user-manual/shardingsphere-jdbc/configuration/yaml/mix.en.md [hunk 2]
```diff
@@ -3,4 +3,67 @@ title = "Mixed Rules"
...
+        props:
+          algorithm-expression: t_user_encrypt_calcite_sharding_${user_id % 2}
+  - !ENCRYPT
+    encryptors:
+      encryptor_aes:
+        type: aes
+        props:
+          aes-key-value: 123456abc
+    tables:
+      t_user_encrypt_calcite:
```
Comment (tristaZero) [inline docs/document/content/user-manual/shardingsphere-jdbc/configuration/yaml/mix.en.md:42]: 2. Could you give `t_user_encrypt_calcite` some explanations?

File: docs/document/content/user-manual/shardingsphere-jdbc/configuration/yaml/mix.en.md [hunk 3]
```diff
@@ -3,4 +3,67 @@ title = "Mixed Rules"
...
+        primaryDataSourceName:
+          - calcite_jdbc_1
+        replicaDataSourceNames:
+          - calcite_jdbc_2
+        loadBalancerName: roundRobin
+    loadBalancers:
+      roundRobin:
+        type: ROUND_ROBIN
+props:
+  sql-show: true
```
Comment (tristaZero) [inline docs/document/content/user-manual/shardingsphere-jdbc/configuration/yaml/mix.en.md:67]: 3. These props are optional, right? Could we point users that they can get more information on xxxx?

File: docs/document/content/user-manual/shardingsphere-jdbc/configuration/yaml/mix.en.md [hunk 4]
```diff
@@ -3,4 +3,67 @@ title = "Mixed Rules"
 weight = 6
 +++
 
-TODO
+* Configuration Item Explanation
+
```
Comment (tristaZero) [inline docs/document/content/user-manual/shardingsphere-jdbc/configuration/yaml/mix.en.md:7]: Hi, 

I suppose we need some introduction about this new section. At least we need to tell users, 
Hi, this is a mixed configuration example, and you can leverage all the features togather.
In order to xx, this page xxxx
Still, to scan the corresponding pages to learn more about the specific feature configuration.xxx

**Review by tristaZero — CHANGES_REQUESTED** [review]
Hi @lmhmhl ,

Here is the primary content, still we need more explanation to guide users more. As you know, what you write will be read by all of the users. Do you think a newcomer will get your point simply?

**Comment by terrymanu** [comment]
I have add an example on #9285, please have a look.
I create the issue #9286 for all related document.

**Comment by lmhmhl** [comment]
> @lmhmhl
> I have add an example on #9285, please have a look.
> I create the issue #9286 for all related document.
@terrymanu 
Thank you, I will have a look, and modify the related document as soon as possible.

**Comment by lmhmhl** [comment]
@terrymanu 
Screen Shot 2021-02-03 at 1 58 44 PM
The sharding algorithm  is not the same name, may be I can change it in my next PR.

**Comment by terrymanu** [comment]
Duplicate with #9289

**Pull request closed without merging**